src/_P088_Ping.ino

```diff
@@ -41,9 +41,31 @@ public:
   P088_data_struct() {
     destIPAddress.addr = 0;
     idseq = 0;
+    if (!P088_data) {
+      P088_data = new P088_icmp_pcb();
+      P088_data->m_IcmpPCB = raw_new(IP_PROTO_ICMP);
+      raw_recv(P088_data->m_IcmpPCB, PingReceiver, NULL);
+      raw_bind(P088_data->m_IcmpPCB, IP_ADDR_ANY);
+      P088_data->instances = 1;
+    } else {
+      P088_data->instances++;
+    }
   }
+
+  ~P088_data_struct() {
+      if (P088_data != nullptr) {
+        P088_data->instances--;
+        if (P088_data->instances == 0) {
+          raw_remove(P088_data->m_IcmpPCB);
+          delete P088_data;
+          P088_data = NULL;
```

Comment:
Please assign a `nullptr` to `P088_data`

Reply:
Done